cocos/rendering/custom/pipeline.ts

```diff
@@ -597,6 +721,10 @@ export interface BasicPipeline extends PipelineRuntime {
      * @zh 结束管线构建
      */
     endSetup (): void;
+    /**
+     * @en Enable cpu culling of objects affected by the light. Enabled by default.
+     * @zh 光照计算时，裁切受光源影响的物件。
```

Comment:
`Enabled by default` is not mentioned in Chinese.

Reply:
fixed